ch_backup/logic/lock_manager.py

```diff
@@ -44,18 +45,29 @@ def __init__(self, lock_conf: dict, zk_ctl: Optional[ZookeeperCTL]) -> None:
         self._disabled = False
         self._lock_timeout = lock_conf.get("lock_timeout")
 
-    def __call__(self, distributed: bool = True, disabled: bool = False):  # type: ignore
+    def __call__(self, distributed: bool = True, disabled: bool = False, lock_id: Optional[str] = None):  # type: ignore ## add 3rd parameter with log info (dict)
         """
         Call-method for context manager
         """
         self._distributed = distributed
         self._disabled = disabled
+        self._lock_id = lock_id
         return self
 
+    def _log_msg_with_lock_id(self, msg):
+        """
+        Append the lock id to a log message, if it is not null
+        """
+        if self._lock_id:
+            return f"{msg} Lock id is {self._lock_id}."
```

Comment:
We use loguru libarary in the project. Consider to use [opt()](https://loguru.readthedocs.io/en/stable/api/logger.html#loguru._logger.Logger.opt) facility and save new logger as an instance variable.

Reply:
Agree, we can define new logger instance in config for such case.

Reply:
OK I'll try to create this new logger. What name should we use?

Reply:
sorry I have missed this comment, and I have implemented a possible solution. I will keep it in a separate commit so it will be easy to revert if it is not good enough